source/PlayniteSDK/Database/IGameDatabase.cs

```diff
@@ -202,14 +202,16 @@ public interface IGameDatabase
         /// </summary>
         /// <param name="game"></param>
         /// <param name="filterSettings"></param>
+        /// /// <param name="nameFilterSearchWithAcronyms"></param>
         /// <returns></returns>
-        bool GetGameMatchesFilter(Game game, FilterPresetSettings filterSettings);
+        bool GetGameMatchesFilter(Game game, FilterPresetSettings filterSettings, bool nameFilterSearchWithAcronyms = false);

```

Comment:
This is breaking change. You need to add new separate method overload.

Reply:
Fixed in https://github.com/JosefNemec/Playnite/pull/3202/commits/17b0389436ad6c5e08e9624bbacf71e8b5e7c611